src/output.cpp

```diff
@@ -1832,7 +1841,7 @@ static void do_kw_subst(chunk_t *pc)
                   unc_text nl_txt;
                   nl_txt.append("\n");
                   nl_idx++;
-                  while ((nl_idx < (size_t)idx) && !unc_isalnum(pc->str[nl_idx]))
+                  while ((nl_idx < idx) && !unc_isalnum(pc->str[nl_idx]))
```

Comment:
maybe my memory is failing but didn't @gmaurel did a lot of PRs switching from `int` to `size_t` and now it appears that you reverted his changes?